.prettierrc.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    trailingComma: 'es5',
+    tabWidth: 4,
+    semi: true,
+    singleQuote: true,
```

Comment:
Do we have a established format for this? 

I like the standard of double quotes (and would simplify to read the diff of this PR), but if there's already a pattern, we can go for this. 
Prettier has pretty good defaults, and IMHO when we go for alternatives it should be well documented.